mkdocs.yml

```diff
@@ -1,22 +1,24 @@
 site_name: Skipper
-pages:
-    - Architecture: index.md
-    - Basics:
-        - Filters: filters.md
-        - Predicates: predicates.md
-        - Scripts: scripts.md
-        - Plugins: plugins.md
-    - Deployments: deployments.md
-    - Data Clients:
-        - Eskip: dataclients/eskip-file.md
-        - Etcd: dataclients/etcd.md
-        - Innkeeper API: dataclients/innkeeper-api.md
-        - Kubernetes: dataclients/kubernetes.md
-        - Route String: dataclients/route-string.md
+nav:
+    - Introduction: index.md
+    - Reference:
+        - Filters: reference/filters.md
+        - Predicates: reference/predicates.md
+        - Scripts: reference/scripts.md
+        - Plugins: reference/plugins.md
+        - Architecture: reference/architecture.md
+        - Development: reference/development.md
+        - Video - How to build: tutorials/video-howto-build.md
+        - Data Clients:
+            - Eskip File: data-clients/eskip-file.md
```

Comment:
Since the directory was renamed the links to it are broken now. In the `deployment.md` file.

Reply:
thanks @arjunrn @szuecs . fixed. Can you please have another look?

Reply:
Looks good now